### PR TITLE
Issue templates

### LINKED ISSUE #6: Feature: Refactor GitHub Issue templates
**Is your feature request related to a problem? Please describe.**
`Bug Report` template is duplicated. `Custom` one - don't even know what it is.

**Describe the solution you'd like**

- Remove duplicated templates,
- Remove `Custom` template,
- Add `Test Request` (To Be Consulted),
- Add `Refactor Request` (To Be Consulted),

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by marekksiazek**
## Summary

Refactor issue templates

## Details

Remove double bug_report file. Add two new files (refactor_request and test_request)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: "[...] improve our code base."

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: "Refactor Request"

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+about: Create a report to help us improve
+title: ""
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: "Refactor: here describe request shortly"

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+labels: ""
+assignees: ""
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: @StraykerPL

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+_File list to refactor_
+
+1.
+2.
+3.
+4.
+
+### Reason of refactoring
+
+_improve the design, structure and implementation of software or improves code readability and reduces complexities or else_
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: "of software or improve [...] and reduce complexity etc.

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+assignees: ""
+---
+
+# Refactor request
+
+### Which files are to refactor
+
+_File list to refactor_
+
+1.
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: Change to bulleted list.

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: "Test Request"

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+about: Create a report to help us improve
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: "Request to perform testing on dev/test environments."

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+title: ""
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: "Test: here describe shortly what is to be tested"

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+labels: ""
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: "testing"

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+assignees: ""
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: @StraykerPL

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+# Test request
+
+---
+
+### Feature author
+
+Author:
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: No need for "Author:" word, just leave space for tagging.

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+---
+
+### Feature description
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: Change to "Tickets involved" and make space for tagging tickets.

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+Featureescription:
+
+---
+
+### What should be the result
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: Bugs will be the results :) No need for this section

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+---
+
+### Tips
+
+Tips on what to pay more attention to when testing:
+
+---
+
+### Requirements to run
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: Better to write "Reproduction steps (optional)" here with numbered list to describe how to reproduce eg. bug showup.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Review by marekksiazek — COMMENTED** [review]
Corrected

File: .github/ISSUE_TEMPLATE/refactor_request.md (hunk 3 above)
Comment (StraykerPL) [reply .github/ISSUE_TEMPLATE/refactor_request.md]: This comment was marked as resolved, but no change was made.

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
+---
+name: Refactor request
+about: Create a report to help us improve our code base
+title: ""
+labels: ""
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: Add "refactor" label

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 17]
```diff
@@ -0,0 +1,22 @@
...
+assignees: StraykerPL https://github.com/StraykerPL
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: Just nickname, no need for URL

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 18]
```diff
@@ -0,0 +1,22 @@
...
+refactor: here describe request shortly
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: This is not right, please delete and move it to "title".

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 19]
```diff
@@ -0,0 +1,22 @@
...
+
+_File list to refactor_
+
+ - ...
+ - ...
+ - ...
+
+### Reason of refactoring
+
+_improve the design, structure and implementation of software or improve code readability and reduces complexitiy etc._
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: "[...]reduce[...]"

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 20]
```diff
@@ -0,0 +1,44 @@
+---
+name: Test Request
+about: Request to perform testing on dev/test environments.
+test: here describe shortly what is to be tested
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: This should be under "title" variable.

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 21]
```diff
@@ -0,0 +1,44 @@
...
+labels: testing
+assignees: StraykerPL https://github.com/StraykerPL
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: Just nickname.

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 22]
```diff
@@ -0,0 +1,44 @@
...
+  2.    
+  3.    
+  4.    
+  5.    
+
+---
+
+### Requirements to run
+
+Requirements:
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: So what exactly will be these "requirements"?

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Review by marekksiazek — COMMENTED** [review]
Corrected

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 23]
```diff
@@ -0,0 +1,21 @@
+---
+name: Refactor request
+about: Create a report to help us improve our code base
+title: Here describe request shortly
+labels: "Refactor"
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: Just "refactor" word, just like issue labels on GitHub

File: .github/ISSUE_TEMPLATE/refactor_request.md [hunk 24]
```diff
@@ -0,0 +1,21 @@
+---
+name: Refactor request
+about: Create a report to help us improve our code base
+title: Here describe request shortly
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/refactor_request.md]: This all line should be: "title: Refactor: Here describe request shortly"

File: .github/ISSUE_TEMPLATE/test_request.md [hunk 25]
```diff
@@ -0,0 +1,38 @@
+---
+name: Test Request
+about: Request to perform testing on dev/test environments.
+title: Here describe shortly what is to be tested
```
Comment (StraykerPL) [inline .github/ISSUE_TEMPLATE/test_request.md]: Just like in refactor template file: "title: Test: Here describe shortly what is to be tested"

**Review by StraykerPL — CHANGES_REQUESTED** [review]

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**